likeminds-chat-reactnative-integration/ChatSX/components/ParseDeepLink/index.ts

```diff
@@ -40,7 +39,7 @@ export async function parseDeepLink(
           uuid: request?.uuid,
           isGuest: false,
         };
-        const initiateUserResponse = await myClient?.initiateUser(
+        const initiateUserResponse = await Client.myClient?.initiateUser(
```

Comment:
chatroom is a file containing methods which are exported from there, so if we declare const myClient = Client.myClient; globally, flow will never go there since its not a component and hence myClient inside methods will be undefined